README.md

```diff
@@ -1,2 +1,113 @@
-# mathMagicians
-This is the first project of the Math Magicians application. I will set up the environment and tools needed to develop a React application. In the following projects, I will develop the actual application.
+<br />
```

Comment:
- [ ] Please ensure that this Pull Requset is following the correct GitHub flow, as informed in the previous review. Kindly follow this [guide]() ohttps://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.mdr watch this [video](https://youtu.be/d0WEe8xH5mk)

Reply:
I have two branches(``Production (default)`` and ``setup``).
I don't understand where is my issue?
Could you provide me feedback?

Reply:
Please make sure to follow correctly GitFlow as specified in the Project Requirements. This one says that they should always be two branches in the repository. A `main` branch and a `development` branch. Make sure that for every feature you develop you open a pull request from the feature branch to `development`. [You can read more about it here.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)

Reply:
At this point you should have `main` and `development` (default) branch. Aside from this two, this pull request should be from `setup` to `development`. So an additional branch for each feature.